types/xrm/index.d.ts

```diff
@@ -3210,6 +3220,12 @@ declare namespace Xrm {
              */
             getControlType(): ControlQuickFormType;
 
+            /**
+             * Gets a reference to the Section parent of the control.
+             * @returns The parent Section.
+             */
+            getParent(): Section;
```

Comment:
Is this no longer needed, or shouldn't have been there to begin with?

Reply:
Added `getParent(): Section` based on the documentation for QuickForm control:

> getParent | Returns a reference to the section object that contains the control.

Reply:
Oh sorry... Saw it as a deletion... Need to check my glass.